src/main/java/com/suchocki/parkingmeter/dao/DriverDAOImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.suchocki.parkingmeter.dao;
+
+import java.util.List;
+
+import org.springframework.stereotype.Repository;
+
+import com.suchocki.parkingmeter.database.FakeDatabaseStub;
+import com.suchocki.parkingmeter.entity.Driver;
+
+@Repository
+public class DriverDAOImpl implements DriverDAO {
+
+	@Override
+	public void save(Driver driver) {
+		FakeDatabaseStub.drivers.add(driver);
+	}
+
+	@Override
+	public Driver get(String id) {
```

Comment:
`Driver` nie ma pola `id`, ma za to `licensePlate` - dlaczego więc tu nazywa się `id`?